client/components/CreateDiscussion/CreateDiscussion.tsx

```diff
@@ -30,22 +32,22 @@ type CreateDiscussionFormValues = {
 };
 
 type CreateDiscussionProps = {
+	parentMessageId: IMessage['_id'];
 	onClose: () => void;
 	defaultParentRoom?: IRoom['_id'];
-	defaultParentRoomName?: string;
+	nameSuggestion?: string;
 };
 
-/*
- ** TO-DO: For other discussion creation locations, some default props should be provided
- ** such as the parent room or parent message. This component does not support this
- ** behavior at this time, but should be implemented when needed.
- */
-
-const CreateDiscussion = ({ onClose, defaultParentRoom }: CreateDiscussionProps): ReactElement => {
+const CreateDiscussion = ({
+	onClose,
+	defaultParentRoom,
+	parentMessageId,
+	nameSuggestion,
+}: CreateDiscussionProps): ReactElement => {
 	const t = useTranslation();
 
 	const { values, handlers } = useForm({
-		name: '',
+		name: nameSuggestion || '',
```

Comment:
If the message is a quote, we receive additionally the link as a suggestion:
![image](https://user-images.githubusercontent.com/27704687/123839946-2338cc00-d8e4-11eb-894b-a63f51795e86.png)

Reply:
This is the same behavior of the old modal. I agree it's not pretty, but there could be anything on the message, we just take the raw text and clip it to a set character count.